phylogenetic/defaults/config.yaml

```diff
@@ -1,7 +1,14 @@
 # Sequences must be FASTA and metadata must be TSV
 # Both files must be zstd compressed
-sequences_url: "https://data.nextstrain.org/files/workflows/zika/sequences.fasta.zst"
-metadata_url: "https://data.nextstrain.org/files/workflows/zika/metadata.tsv.zst"
+inputs:
+  - name: ncbi
+    metadata: "s3://nextstrain-data/files/workflows/zika/metadata.tsv.zst"
+    sequences: "s3://nextstrain-data/files/workflows/zika/sequences.fasta.zst"
+
+additional_inputs:
+  - name: usvi
+    metadata: "data/metadata_usvi.tsv"
+    sequences: "data/sequences_usvi.fasta"
```

Comment:
[this comment thread assumes we want to merge USVI in phylo, see other thread discussing this]

If USVI is defined via `additional_inputs` then we have the following situations:
* If a config overlay defines `additional_inputs` then it'll clobber USVI. I don't think we want this.
* If a config overlay defines `inputs`, the intention is that i'll replace the defaults. However since we have `additional_inputs` USVI will be added to the user's `inputs`, which is unexpected.

I think we instead want to define USVI as a second entry within `inputs`

Reply:
> I think we instead want to define USVI as a second entry within inputs
+1 makes sense